app/views/admin/users/index/controls/_export.html.erb

```diff
@@ -0,0 +1,5 @@
+<a href="<%= export_admin_users_path %>.csv">
```

Comment:
I didn't spend too much time on this because at the moment the icon is enclosed by a link, however in the next PR, the icon will instead behave as a button since we first show a confirmation modal.

Reply:
Like you say we can handle it in a later PR, but we should be sure to export the SVG at the usual crayons sizes (24 x 24) etc and add it as a reusable asset